packages/x-charts/src/BarChart/BarPlot.tsx

```diff
@@ -97,22 +98,42 @@ const useCompletedData = (): CompletedBarData[] => {
       if (verticalLayout) {
         if (!isBandScaleConfig(xAxisConfig)) {
           throw new Error(
-            `Axis with id "${xAxisKey}" shoud be of type "band" to display the bar series of id "${seriesId}"`,
+            `MUI-Charts: ${
```

Comment:
Nitpick: Have you considered going with a `MUI-X-Charts` prefix instead to be more explicit/correct? 🤔

Reply:
No, but it's a good idea :)